src/caret_analyze/architecture/architecture.py

```diff
@@ -356,6 +356,21 @@ def _verify(nodes: Collection[NodeStruct]) -> None:
 
     def update_message_context(self, node_name: str, context_type: str,
                                subscribe_topic_name: str, publish_topic_name: str) -> None:
+        """
+        Update message_context of node_path in "node_name" node.
+
+        Parameters
+        ----------
+        node_name : str
+            name of target node to update message_context
+        context_type : str
+            type name of updating message_context
+        subscribe_topic_name : str
+            name of subscribe topic of node_path updating message_context
+        publish_topic_name : str
+            name of publish topic of node_path updating message_context
```

Comment:
```suggestion
            name of publish topic of node_path updating for message_context
```

Reply:
Fix it in [c5a7bd9](https://github.com/tier4/CARET_analyze/pull/300/commits/c5a7bd978be6ff8d12c907120a79795467a95842).